### PR TITLE
Timeseries update

### PR DESCRIPTION
**Pull request by thenaomig**
This PR addressed two outstanding loose ends in timeseriestools.py
https://trello.com/c/g35VOHPb
and
https://trello.com/c/LRwiznwG

- You can no longer select a resampling window smaller than the time resolution the data is starting with
- You can select min, max, and/or percentile all at once instead of only one at a time.

### CODE REVIEW

**Comment by thenaomig** [comment]
Something I changed is incompatible with how the test is written... looking into it.

**Review by nicolejkeeney — CHANGES_REQUESTED** [review]
The way you have it coded now will cause an error if the following line of code is run more than once in a notebook: 

```timeseries = tst.Timeseries(my_data)``` 

The error it raises: 
error

Looking at the backend code, I think this is because the dictionary `_time_scales` is deleting the keys for days and hourly the first time the `Timeseries` class is used.

The first time I ran this it worked great, though!

**Comment by thenaomig** [comment]
> The way you have it coded now will cause an error if the following line of code is run more than once in a notebook:
> 
> `timeseries = tst.Timeseries(my_data)`

Should be resolved with latest update.

**Review by nicolejkeeney — APPROVED** [review]
Looks good to me!

**Review by vicford — APPROVED** [review]
This looks great - I really like the compound min/mean/percentile display

**Review by elehmer — APPROVED** [review]
Works great! Just the one code reduction comment...

File: climakitae/timeseriestools.py [hunk 1]
```diff
@@ -15,30 +15,43 @@ class _TimeSeriesParams(param.Parameterized):
...
+    _time_scales = dict(
+        [("hours", "H"), ("days", "D"), ("months", "MS"), ("years", "AS-SEP")]
+    )
+
     def __init__(self, dataset, **params):
         super().__init__(**params)
         self.data = dataset
 
+        _time_resolution = dataset.attrs["frequency"]
+        if _time_resolution == "monthly":
```
[the remark this answers is not in the document]
Comment (elehmer) [reply climakitae/timeseriestools.py]: Awesome!

**Pull request merged**